docs/tutorial.md

```diff
@@ -97,18 +105,27 @@ ordering to define tiers of services.  In this example, we want all
 
 ```java
 private static final String TASK_METADATA_NAME = "meta-data";
-private static final int TASK_METADATA_COUNT = Integer.valueOf(System.getenv("METADATA_COUNT"));
-private static final double TASK_METADATA_CPU = Double.valueOf(System.getenv("METADATA_CPU"));
-private static final double TASK_METADATA_MEM_MB = Double.valueOf(System.getenv("METADATA_MEM"));
-private static final double TASK_METADATA_DISK_MB = Double.valueOf(System.getenv("METADATA_DISK"));
-private static final String TASK_METADATA_URI = System.getenv("METADATA_URI");
+private static final int TASK_METADATA_COUNT =
```

Comment:
Why split up these lines?  They aren't split in the code, and doing so would be non-standard Java.